Training/python/feature_scaling.py

```diff
@@ -57,11 +61,13 @@
         raise ValueError('Specified file_range is not valid: should be either -1 (run on all files in file_path) or range [a, b] with a<=b')
     file_names_id = [fname.split('_')[-1].split('.root')[0] for fname in file_names] # id as taken from the name: used as file identifier (key field) in the output json files with quantiles
     #
-    dR_tau_outer_cone = cone_definition_dict['outer']['dR']
-    tau_pt_name, tau_eta_name, tau_phi_name = cone_selection_dict['TauFlat']['var_names']['pt'], cone_selection_dict['TauFlat']['var_names']['eta'], cone_selection_dict['TauFlat']['var_names']['phi']
-    inner_cone_min_pt = cone_definition_dict['inner']['min_pt']
-    inner_cone_min_radius = cone_definition_dict['inner']['min_radius']
-    inner_cone_opening_coef = cone_definition_dict['inner']['opening_coef']
+    # if cone_definition_dict != None:
```

Comment:
feature_scaling should work for both reco and id. please add cones as an option and uncomment the code

Reply:
@kandrosov  This code is already compatible with both ID and RECO configs, these comments are more just to see the difference when reviewing. But may be a bit tricky to follow the logic, I used debugger for this. If needed, I can reshuffle code a bit, but I do not have ideas how to do it on-flight.

Reply:
About in particular these variables: they are somehow hidden inside scaling_utils.py(fill_aggregators) and addressed by demand